molecule/default/molecule.yml

```diff
@@ -5,25 +5,31 @@ dependency:
 driver:
   name: docker
 lint:
-  name: yamllint
   enabled: false
-  options:
-    config-file: .yamllint.yml
 platforms:
   - name: centos-6
-    image: docker.io/linuxsystemroles/centos-6
+    image: registry.centos.org/centos:6
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
     privileged: true
+    command: /sbin/init
   - name: centos-7
-    image: docker.io/linuxsystemroles/centos-7
+    image: registry.centos.org/centos/systemd:latest
```

Comment:
The latest is centos8, isn't it? But I do not seem to be able to access this registry for some reason.

Reply:
> The latest is centos8, isn't it? But I do not seem to be able to access this registry for some reason.

The `centos/systemd` is a special centos7 image built with systemd enabled - there isn't one for centos8 because the centos8 base image has systemd enabled.